models/i18n.cfc

```diff
@@ -165,86 +165,95 @@ component singleton accessors="true" {
 	 *
 	 * @thisLocale Locale to validate
 	 */
-	boolean function isValidLocale( required string thisLocale ) {
-		return ( listFind( arrayToList( getLocales() ), arguments.thisLocale ) ) ? true : false;
+	boolean function isValidLocale( required string thisLocale ){
+		return (
+			listFind(
+				arrayToList( getLocales() ),
+				arguments.thisLocale
+			)
+		) ? true : false;
 	}
 
 	/**
 	 * returns array of locales
 	 */
-	array function getLocales() {
+	array function getLocales(){
 		return variables.aLocale.getAvailableLocales();
 	}
 
 	/**
 	 * returns list of locale names, UNICODE direction char (LRE/RLE) added as required
 	 */
-	string function getLocaleNames() {
-		var orgLocales   = getLocales();
+	string function getLocaleNames(){
 		var theseLocales = "";
 		var thisName     = "";
-		orgLocales.each( function( orgLocale ) {
-			if ( orgLocale.listLen( "_" ) == 2 ) {
+		for ( orgLocale in getLocales() ) {
```

Comment:
Why change this from the original `each`? This actually introduces a var scope issue with `orgLocale`.  Can we revert back.

Reply:
orgLocales is actually getLocales which is a Java array, not a cf array. I like the cfarray.each syntax more, but I had many places where in  the code where I got undocumented responses from the original java array so I didn't want to change it anymore. Things like
```
	boolean function isValidLocale( required string thisLocale ){
		return (
			listFind(
				arrayToList( getLocales() ),
				arguments.thisLocale
			)
		) ? true : false;
	}
```
or this
```
		for ( var orgLocale in getLocales() ) {
			if ( listLen( orgLocale, "_" ) == 2 ) {
```
which had different results (or errors) if I changed getLocales result to a cf array. 
There are many more places where java arrays are returned, but I never heard anyone complain about that.
So I leave it for now, or we should further rewrite the module.

Reply:
I also rewrote getLocaleQuery() because it was broken (but not in tests)